db/version_set.cc

```diff
@@ -4694,6 +4695,84 @@ Status VersionSet::ReduceNumberOfLevels(const std::string& dbname,
       mutable_cf_options, &ve, &dummy_mutex, nullptr, true);
 }
 
+// Get the checksum information including the checksum and checksum method
+// name of all SST files of this Manifest. Store the information in
+// FileChecksumLis which contains a map from file id to its checksum info
+Status VersionSet::GetAllFileCheckSumInfo(Options& options,
```

Comment:
Do you need to update `VersionSet::GetLiveFilesMetaData()`, too?

Reply:
@riversand963 Yes, thanks.